hal/armv7m/imxrt/117x/peripherals.h

```diff
@@ -242,7 +242,7 @@
 #define FLASH_FLEXSPI1_INSTANCE  0x1
 #define FLASH_FLEXSPI1_QSPI_FREQ 0xc0000007
 
-#define FLASH_FLEXSPI2_MOUNTED   0
+#define FLASH_FLEXSPI2_MOUNTED   1
```

Comment:
Is it mounted? Side note - perhaps this should be moved to board_config.h (not necessarily in this PR)

Reply:
Hmm, there is hyperflash NOR on evalkit connected to flexspi2, which I tested I will turn it off (not mount) for now as mostly we'll not use it, we'll use only serial spi NOR which is connected to flexspi1 controller.